packages/react/src/patterns/sub-patterns/ContentGroup/ContentGroup.js

```diff
@@ -20,41 +21,47 @@ const { prefix } = settings;
  * @param {object} props props object
  * @param {*} props.children JSX Components
  * @param {string} props.heading  Heading objects
+ * @param {string} props.className optional class to be applied to the containing node
+ * @param {*} props.cta CTA component props object
  * @returns {*} JSX ContentArrayGroup component
  */
-const ContentGroup = ({ children, heading }) => {
+const ContentGroup = ({
+  children,
+  heading,
+  className: customClassName,
+  cta,
+}) => {
+  const className = classNames(`${prefix}--content-group`, {
+    [customClassName]: !!customClassName,
+  });
+
   return (
-    <div
-      className={`${prefix}--content-group`}
-      data-autoid={`${stablePrefix}--content-group`}>
-      <div className={`${prefix}--content-group__container`}>
-        <div className={`${prefix}--content-group__row`}>
-          <div className={`${prefix}--content-group__col`}>
-            <h3
-              data-autoid={`${stablePrefix}--content-group__title`}
-              className={`${prefix}--content-group__title`}>
-              {heading}
-            </h3>
-          </div>
-        </div>
-        <div className={`${prefix}--content-group__row`}>
-          <div
-            data-autoid={`${stablePrefix}--content-group__children`}
-            className={classNames(
-              `${prefix}--content-group__col`,
-              `${prefix}--content-group__children`
-            )}>
-            {children}
-          </div>
-        </div>
+    <div className={className} data-autoid={`${stablePrefix}--content-group`}>
+      <h3
+        data-autoid={`${stablePrefix}--content-group__title`}
+        className={`${prefix}--content-group__title`}>
+        {heading}
+      </h3>
+      <div
+        data-autoid={`${stablePrefix}--content-group__children`}
+        className={classNames(
+          `${prefix}--content-group__col`,
+          `${prefix}--content-group__children`
+        )}>
+        {children}
+      </div>
+      <div className={`${prefix}--content-group__CTA`}>
+        <CTA style="card" type="local" {...cta} />
```

Comment:
Is this optional? Do we ever render null if the `CTA` is no available?

Reply:
I think it's working, but I think it's working based on possibly legacy stuff within the card? Does it work if we change it to a text type link? Do you have the latest work from Anna's card PR?